packages/curiefense/generated-changes/patch/Chart.yaml.patch

```diff
@@ -0,0 +1,15 @@
+--- charts-original/Chart.yaml
++++ charts/Chart.yaml
+@@ -2,4 +2,10 @@
+ appVersion: 1.0.0
+ description: Complete curiefense deployment
+ name: curiefense
+-version: 1.0.0
+\ No newline at end of file
++version: 1.0.0
++
++kubeVersion: # A SemVer range of compatible Kubernetes versions. E.g 1.18 - 1.21, >= 1.19, etc
```

Comment:
You need to set the value of `kubeVersion` to a semver range of K8s versions that are compatible with your chart. For example, if your charts is compatible with k8s v1.18 and above you would set this to `kubeVersion: ">= 1.18"`.